packages/api/lambdas/bulk-operation.js

```diff
@@ -162,9 +130,10 @@ async function bulkGranuleDelete(
  * @returns {Promise}
  */
 async function bulkGranule(payload) {
+  const knex = await getKnexClient();
   const { queueUrl, workflowName, meta } = payload;
   const granuleIds = await getGranuleIdsForPayload(payload);
-  return await applyWorkflowToGranules({ granuleIds, workflowName, meta, queueUrl });
+  return await applyWorkflowToGranules({ knex, granuleIds, workflowName, meta, queueUrl });
```

Comment:
Nit: sort/formatting

Reply:
👍 
https://github.com/nasa/cumulus/pull/2415/commits/48a7b5439f76706e936806a698a62ba42225080a

I left `knex` at the top because it's often the first argument in our model methods and helpers. Happy to fight about it though 😄